tianshou/trainer/offpolicy.py

```diff
@@ -104,13 +104,13 @@ def offpolicy_trainer(
                             train_fn(epoch)
                 for i in range(update_per_step * min(
                         result['n/st'] // collect_per_step, t.total - t.n)):
-                    global_step += 1
+                    global_step += collect_per_step
                     losses = policy.update(batch_size, train_collector.buffer)
                     for k in result.keys():
                         data[k] = f'{result[k]:.2f}'
                         if writer and global_step % log_interval == 0:
-                            writer.add_scalar(
-                                k, result[k], global_step=global_step)
+                            writer.add_scalar('train/' + k, result[k],
```

Comment:
Nice addition! Hierarchical data logging is very useful when a lot of data is monitored.